tests/func/test_gc.py

```diff
@@ -204,3 +205,15 @@ def test_gc_no_dir_cache(tmp_dir, dvc, repo_template):
 
 def _count_files(path):
     return sum(len(files) for _, _, files in os.walk(path))
+
+def test_gc_no_unpacked_dir(tmp_dir, dvc, repo_template):
+    dir_stages = dvc.add("dir")
```

Comment:
you can use `stage, = tmp_dir.dvc_gen({"dir": {"file": "file_content"}})`

Reply:
if you choose to go this way there will be no need for tepo_template :)

Reply:
@pared `repo_template` is fine too though.